tests/profile/benchmark-line-parsing.c

```diff
@@ -19,6 +21,40 @@
     __hash_value;                                                    \
 })
 
+char *netdata_configured_host_prefix = "";
+
+void netdata_cleanup_and_exit(int ret)
+{
+    exit(ret);
+}
+
+void signals_unblock(void)
+{
+}
+
+void signals_reset(void)
+{
+}
+
+int health_variable_lookup(const char *variable, uint32_t hash, RRDCALC *rc, calculated_number *result)
+{
+    (void)variable;
+    (void)hash;
+    (void)rc;
+    (void)result;
+
+    return 0;
+}
+
+void send_statistics(const char *action, const char *action_result, const char *action_data)
+{
+    (void)action;
+    (void)action_result;
+    (void)action_data;
+
+    return;
+}
```

Comment:
Hi

https://docs.netdata.cloud/contributing/#code-style

Reply:
Hi @ilyam8, I'm not sure which bit of the coding style you're referring to.

- It's using spaces instead of tabs.
- Lines are under 120 chars long.
- Opening curly brace is on a new line, coding style says you're *allowed* not *should/must* to put it on the same line, I've seen it both ways in the code, including this file in question and kernel coding style is to put it on the same line except for functions (because K&R says so and you can't nest functions in ANSI C).
- No space between cast and variable, AFAICS neither netdata nor kernel coding style make a comment on this and I've seen it both ways in both code bases.

Any hints?

Yes, there is a lot of coding style inconsistencies in the code base, even just in this file there is inconsistent indentation, curly brace placement, space or no space between casts and variables & lines longer than 120 chars.

At least once my __always_unused/__maybe_unused patch is accepted, all that casting of variables in the dummy functions will go.

Cheers,
Andrew

Reply:
I guess the main thing is `The single most important rule when writing code is this: check the surrounding code and try to imitate it`. This file is quite inconsistent in that regard, it's true that many files in netdata use different styles.

Reply:
> Opening curly brace is on a new line

Mainly this.

But yes, @mfundul is right.

Reply:
That is how it was, i openned the PR and noticed that surrounded code is not 

> Opening curly brace is on a new line

Reply:
This is a comment, not a request to change.

Reply:
Yeah, the file was already inconsistent.

Reply:
I'll take another look st the files in there and try and at least make the curly brace placement consistent.